treefmt.nix

```diff
@@ -6,21 +6,23 @@
     { pkgs, ... }:
     {
       treefmt = {
+        flakeCheck = pkgs.hostPlatform.system != "riscv64-linux";
```

Comment:
Why not use availableOn pattern?

Reply:
I think that broke at some point and currently it's a dependency deep in the dependency tree of treefmt, which renders riscv64-linux unusable until the next staging cycle.